app/models/backend_api_routing_policy_injector.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+class BackendApiRoutingPolicyInjector
```

Comment:
Q1: Are we using later this same class to merge all proxy rules of each backend (prepending the backend path) into the proxy config?

if so...
Q2: Can we also use it when showing/counting proxy rules of a service at https://github.com/3scale/porta/blob/9bcb8724c6931d8dd2f0b148c14f193fcc4b4eec/app/presenters/api/integrations_show_presenter.rb#L58?

Reply:
Q1: It is only used in `Proxy#policy_chain`, when saving the configuration to the proxy config.
Q2: I do not understand